CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Contribute to Flutter docs
+description: Ways to contribute to the Flutter project.
+---
+
+If you would like to help contribute to the Flutter project,
+we’re happy to have your help! Anyone can contribute, whether you’re new to
+the project or you’ve been around a long time, and whether you self-identify
+as a developer, an end user, or someone who just can’t stand seeing typos.
+
+For more ways to get involved in the Flutter community or to learn about us,
+also visit the [Flutter community](/community) page.
+
+Looking for the [Effective Dart]({{site.www}}/guides/language/effective-dart)?
+This comprehensive set of guides for the Dart language (which is used by Flutter),
+includes guides on style, documentation, usage, and design.
+
+# Ways to contribute
+
+We encourage you to reach out and join the conversation in a variety of ways.
+
+An easy way to send feedback is to "thumbs up" issues important to you
+in either the issue tracker for the [Flutter SDK and API docs][issues],
+or the [flutter.io website][doc-issues].
+
+Other ways you can contribute:
+
+* [Ask HOWTO questions that can be answered with specific solutions][so]
+* [Live chat with Flutter engineers and users][gitter]
+* [Discuss Flutter, best practices, app design, and more on our mailing list][mailinglist]
+* [Report bugs and request features][issues]
+* [Report API docs bugs][issues]
+* [Request docs for flutter.io][doc-issues]
+* [Follow us on Twitter: @flutterio](https://twitter.com/flutterio/)
+* [Read the Flutter Publication on Medium](https://medium.com/flutter-io)
+* [Sign up to Future UX Studies on Flutter](/research-signup)
+* [Join the subreddit to keep up with the latest in the Flutter community][reddit]
+
+[issues]: https://github.com/flutter/flutter/issues
+[doc-issues]: https://github.com/flutter/website/issues
+[so]: https://stackoverflow.com/tags/flutter
+[mailinglist]: https://groups.google.com/d/forum/flutter-dev
+[gitter]: https://gitter.im/flutter/flutter
+[reddit]: https://www.reddit.com/r/FlutterDev
+
+For more information on contributing, see
+[Contributing to Flutter](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md)
+in the [Flutter SDK](https://github.com/flutter/flutter) repo.
```

Comment:
this final paragraph is what I would have expected to see near the top. This is the main way I hope to see people contribute: by writing code and pushing PRs.